cli/tauri.js/package.json

```diff
@@ -11,8 +11,8 @@
   },
   "scripts": {
     "build": "yarn build:api && yarn build:typedefs && yarn build:webpack",
-    "build:webpack": "webpack --progress",
-    "build:api": "rollup -c --silent && yarn build:typedefs",
+    "build:webpack": "rm ./dist -R || webpack --progress",
+    "build:api": "rm ./api -R || rollup -c --silent && yarn build:typedefs",
```

Comment:
Should this not be `&&`?

Reply:
if the dist folder doesn't exist we just proceed with the build script, the `rm` is here just to cleanup old artifacts. (it's not really useful on CI because it's always a clean state, but locally it's helpful).